File: BASIC_CODING/TERENARY_OPERATORS_2/solution.js
/**
 * Find and print the maximum between two numbers.
 *
 * @param {number} a - The first number.
 * @param {number} b - The second number.
 */
function findMax(a, b) {
	let result = a > b ? a : b;
	console.log(`Finding the maximum between ${a} and ${b}:`);
	console.log(`Maximum: ${result}`);
}

// Example usage:
let a = 3;
let b = 5;
findMax(a, b);

File: BASIC_CODING/TERENARY_OPERATORS_2/solution.java
public class Main {
    /**
     * Find and print the maximum between two numbers.
     *
     * @param a The first number.
     * @param b The second number.
     */
    public static void findMax(int a, int b) {
        int result = a > b ? a : b;
        System.out.println("Maximum: " + result);
    }

    public static void main(String[] args) {
        int a = 3;
        int b = 5;
        System.out.println("Finding the maximum between " + a + " and " + b + ":");
        findMax(a, b);
    }
}
